packages/icons/icons.json

```
@@ -4198,5 +4198,12 @@
         "tags": "data, type, format，punctuation, grammar",
         "group": "miscellaneous",
         "codepoint": 62296
+    },
+    {
+        "displayName": "Bullseye",
+        "iconName": "bullseye",
+        "tags": "geography, location, position, map, direction, bullseye, reference, point",
```

```suggestion
        "tags": "geography, location, position, map, direction, reference, point, target",
```

You can remove "bullseye" from this list since the search logic will auto-include the name of the icon as well. And then I think it's also worth including "target," since I could see that being searched for.